.gitignore

```diff
@@ -60,6 +60,8 @@ dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/
 
+#personal profiles
+**/application-paco.yml
```

Comment:
As we now have the new approach for personal profiles, I would suggest to remove that yml from the public repo and keep it locally as `application-private-paco.yml`, as suggested in
https://github.com/coatrack/coatrack/pull/163/files